crates/wasm/src/lib.rs

```diff
@@ -17,8 +17,14 @@ pub struct BuildInfo {
     dirty: &'static str,
 }
 
-#[derive(Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize)]
 pub struct WASMCompileOptions {
+    #[serde(default = "default_entry_point")]
+    entry_point: String,
+
+    #[serde(default = "default_circuit_name")]
+    circuit_name: String,
```

Comment:
I assume this is added so that we can prepend it onto the names of compile contract functions similarly to how we handle them in the CLI. If so then we need to add that as this is currently unused.